centreon/doc/API/centreon-api-v23.04.yaml

```
@@ -357,6 +357,11 @@ paths:
         * name
         * alias
         * is_activated
+        
+        Changes in 23.04 :
+        
+        * `icon` is no longer an object but the image ID
+        * `icon_map` is no longer an object but the image ID
```

This information should be updated with new property names
```suggestion
        
        Changes in 23.04 :
        
        * `icon` is rename `icon_id`
        * `icon_map` is rename `icon_map_id`
```